_posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown

```diff
@@ -214,40 +214,42 @@ Verify all the cluster health, indices, settings etc are seamlessly migrated and
+3. Rolling Upgrade: This is the most complex approach, but requires no downtime and provides cluster health feedback at every step of the process. This method takes down one node at a time for an upgrade. We recommend this approach for advanced users.
 
-Now that you read through most of the good stuff, We would recommend these in terms of ease of upgrade:
+### Upgrading OpenSearch Dashboards 
 
-1. Snapshot Upgrade: This is the safest approach if you can spin up multiple clusters and move your customers to the new endpoint after the upgrade. we would recommend this for beginners. 
-2. Restart Upgrade: This is a moderate approach, as this would take down the whole cluster and you will only know after the whole cluster upgrade to verify everything is happy. we would recommend this for intermediate users or for people who know what they are doing.
-3. Rolling Upgrade: This is a complex approach, as this would take down one node at a time and upgrade them. The good thing about this approach is that you get cluster health feedback while going through the process. we would  recommend this for advanced users.
+Similarly to Kibana OSS, OpenSearch Dashboards currently does not support rolling upgrades, requiring a Restart Upgrade. To upgrade to OpenSearch Dashboards, stop all Open Distro Kibana instances, deploy a new OpenSearch Dashboards instance, and direct traffic to it.
 
-### Going forward
+### Going Forward
 
-OpenSearch can be upgraded using a Rolling upgrade process for most minor and major versions so upgrading going forward will not usually interrupt service. Like Kibana OSS, OpenSearch Dashboards does not support rolling upgrades, but it supports restart upgrades. You are able to stop all Open Distro instances, deploy a new OpenSearch Dashboards instance and direct traffic to it.
+OpenSearch can be upgraded using a Rolling Upgrade for most minor and major versions, thus upgrading going forward will not usually interrupt your service.
 
-For more information on upgrading please see our [Upgrading FAQs](https://opensearch.org/faq/#c3) and if you have questions about backward compatibility you can check dB’s post on [Backwards Compatibility FAQ](https://opensearch.org/blog/technical-posts/2021/06/opensearch-backwards-compatibility-faq/). If you’re looking for a great “getting started” guide I’d recommend Gedalyah’s [Install and Configure OpenSearch guide](https://logz.io/blog/opensearch-tutorial-installation-configuration/).
+For more information on upgrading please see our [Upgrading FAQs](https://opensearch.org/faq/#c3). If you have questions about backward compatibility, check out [this blog post](https://opensearch.org/blog/technical-posts/2021/06/opensearch-backwards-compatibility-faq/). If you’re looking for a great "Getting Started" guide, I highly recommend Gedalyah’s [Install and Configure OpenSearch](https://logz.io/blog/opensearch-tutorial-installation-configuration/).
```

Comment:
We should avoid links like 'this blog post' as it bad for SEO and accessibility (screen readers). The link text should describe subject matter of the link.

Reply:
Fixed.